src/__tests__/integration/integration.node.js

```diff
@@ -0,0 +1,53 @@
+/** Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
As of now, there is not a test for React 15.  Open to thoughts on the best way to support that case!